index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio : Mobile version skeleton</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+  </head>
+  <body>
+    <!-- Navbar part   -->
+    
+    <header>
+      <nav>
+        <div class="nav-item brand">John Doe</div>
+        <div class="nav-item menu"><i class="fa-solid fa-bars"></i></div>
+      </nav>
+    </header>
+
+    <!-- Home part -->
+    <section>
+      <h1>Laura Jong</h1>
+      <h2>Software Developer</h2>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <hr />
```

Comment:
- [ ] Kindly use css to add margin or padding this will make your project more semantic, It would be better to remove all `<hr>` tag  from your project 👍🏿